### PR TITLE
Suppress the last two MemorySanitizer failures

### PR DESCRIPTION
**Pull request by akuzm**
Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Suppress some test failures under MemorySanitizer.

### CODE REVIEW

File: dbms/tests/msan_suppressions.txt [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # https://github.com/google/oss-fuzz/issues/1099
 fun:__gxx_personality_*
+
+# We apply std::tolower to uninitialized padding, but don't use the result, so
```
Comment (alexey-milovidov) [inline dbms/tests/msan_suppressions.txt]: Why don't just add suppression in source file via function attribute?
Look at `LZ4_decompress_faster.cpp`.

File: dbms/tests/msan_suppressions.txt [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+# it is OK. Reproduce with "select ngramDistanceCaseInsensitive(materialize(''), '')"
+fun:tolower
+
+# A failure in zlib-ng in 00302_http_compression. Doesn't reproduce locally for me.
```
Comment (alexey-milovidov) [inline dbms/tests/msan_suppressions.txt]: It reproduces perfectly on server.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

**Comment by akuzm** [comment]
The tests still don't pass: https://clickhouse-test-reports.s3.yandex.net/8780/8bf260454e5c69453d4259bccc17d991ccd3b698/functional_stateless_tests_(memory)/stderr.log

**Comment by alexey-milovidov** [comment]
Waiting for normal fix.

File: dbms/tests/msan_suppressions.txt (hunk 1 above)
Comment (akuzm) [reply dbms/tests/msan_suppressions.txt]: Reached a conclusion that this is OK on Telegram.

**Comment by akuzm** [comment]
I merged the least-controversial toLower suppression.

**Pull request merged**